spec/controllers/vacations_controller_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+describe VacationsController do
+  let(:user) { create(:user) }
+
+  before do
+    allow(controller).to receive(:authenticate_user!)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  describe 'GET index' do
+    let!(:vacation) { create(:vacation, user: user) }
+
+    it "returns all user's vacations" do
```

Comment:
this spec is only checking if the request is successful, but the `it` suggests it's testing if all the user's vacations is being returned. I'd suggest renaming this one to reflect what it's testing and then add a new one to check if we're getting the vacations.

Reply:
Could you review again? I just updated these tests